src/rich-text/commands/index.ts

```diff
@@ -421,6 +474,43 @@ const headingDropdown = (schema: Schema) =>
         )
     );
 
+const overflowDropdown = (schema: Schema, options: CommonViewOptions) =>
+    makeMenuDropdown(
+        "Overflow",
+        _t("commands.overflow"),
+        "overflow-dropdown",
+        () => true,
+        () => false,
+        dropdownItem(
+            _t("commands.tag", { shortcut: getShortcut("Mod-[") }),
```

Comment:
Let's standardize on the term "tagLink" here as "tag" could be otherwise confusing.